### DISCUSSION TITLE
Game Gear & TurboGrafx16 Adapters

### DISCUSSION THREAD

**Discussion by kdawg8** [sanni/cartreader#168] [Hardware]
With the Retrode forum down does anyone know where I can find info on building the adapters needed for Game Gear and TurboGrafx 16?  Thanks!

**Comment by PokeyMinch** [comment]
About:Game Gear adapter

You can use retrode2-sms-plugin （adapter） .

https://stoneagegamer.com/master-system-adapter-for-retrode-2-rip-play-sega-master-system-roms.html

Soldering original Game Gear socket on pcb of sms-plugin.

About:PCE/TG16 adapter info

https://forum.arduino.cc/t/rom-reader-for-super-nintendo-super-famicom-game-cartridges/155260/542#msg3729576

Soldering original PCE/TG16 socket or GAMEBANK-web.com original PCE adapter socket.
![image](https://user-images.githubusercontent.com/85623113/121477145-6ea53c00-ca02-11eb-9c53-e1b457e2f44a.jpeg)

**Reply by kdawg8** [reply]
Thanks!  What is the correction orientation for the gg slot on the sms adapter pcb?

**Comment by PokeyMinch** [comment]
There is a space(pinhole) to solder on a Game Gear cartridge connector to this adapter PCB. It’s a piece of cake .
![image](https://user-images.githubusercontent.com/85623113/121797271-339d4580-cc5a-11eb-9396-e2cbac72e0c2.jpeg)
![image](https://user-images.githubusercontent.com/85623113/121797278-3b5cea00-cc5a-11eb-9b78-85c5efbcf960.jpeg)

**Comment by kdawg8** [comment]
Fantastic!  That's much easier than I was thinking it would be.  Any idea where I can buy a game gear cartridge slot though?  I've come up empty on every way I can think of to search for it, only thing I can find is http://www.tototek.com/store/index.php?main_page=product_info&cPath=22&products_id=217 which is out of stock.

**Comment by LNRC** [comment]
Would the new RetroSix Game Gear GameSlot kit also work for the Game Gear adapter?
https://retrosix.co.uk/Game-Gear-GameSlot-Kit-p515558760
![3365081366](https://user-images.githubusercontent.com/39219937/210038548-9fee6f93-fd8f-437d-867d-1a8ecc407a65.jpg)

**Reply by kdawg8** [reply]
It wouldn’t work unfortunately, what the cart reader needs is an adapter to go from a game gear socket to a md/genesis cartridge pcb.  This socket you found would be a nice alternative to desoldering one from a non-working console though.  I’ve made gg adapters for a few for people, send me a pm if you need help getting one built.

**Reply by LNRC** [reply]
Ah my apologies, I meant soldering this cartridge slot _onto_ the Retrode. I don't like taking parts from consoles that aren't totally broken, so I'm hoping this cartridge slot can be an effective alternative!

**Comment by LNRC** [comment]
![PXL_20230105_062432260](https://user-images.githubusercontent.com/39219937/210715775-1f48b3ab-a314-49a0-9bfa-226781c404c0.jpg)
Success with the Retro Six Gameslot GG!

**Reply by PsyK0p4T** [reply]
Very nice to see, congrats :o 
thanks for this test !

**Reply by 7ank0v1c** [reply]
Hey man, loved this so much I went and bought the slot myself. It just arrived today. I am just wondering on the installation process though. Am I right in thinking that there are no pins to solder, just simply lay it flat onto the pcb, and solder the holes, lined up to the pads underneath?

**Reply by 7ank0v1c** [reply]
Also, I actually have a sms retrode cartridge, but was wondering if there is a dedicated gg adapter made yet?

**Reply by kdawg8** [reply]
The recently created “user-contributed pcbs” thread on here is compiling a list of adapter pcbs but there isn’t one for gg so as far as I’m aware the sms adapter is the only one.

You can give it a shot without the solder by holding the pins against the edges of the holes but the socket really should be soldered to the pcb to get a reliable electrical connection on all pins.

**Reply by 7ank0v1c** [reply]
Could you possibly send a photo of how you soldered yours both on back and front of sms retrode pcb by any chance?

**Reply by kdawg8** [reply]
![DE3176A6-CD3B-4002-8BE6-60C8380D5B8A](https://user-images.githubusercontent.com/83562863/211705995-fc42092c-83ae-4368-8847-a086fd7b49d4.jpeg)
![5483729B-EE77-48C0-A821-328A799CEF48](https://user-images.githubusercontent.com/83562863/211706021-34fa1376-05a9-4427-8aa6-812b4c439c85.jpeg)

**Reply by 7ank0v1c** [reply]
That looks like the a lot from an actual GG? I thought you used the GG slot from RetroSix?

**Reply by 7ank0v1c** [reply]
Oh wait, I just realised you aren’t the person who posted the pic with slot from retrosix 😂

**Reply by 7ank0v1c** [reply]
@LNRC Are you able to help me? I also have bought the same GameSlot GG and am trying to use it with my retrode adapter. However it doesn’t seem to work for some reason?

Did you have any issues with it? Or was it a simple quick solder and it worked?

Also, how did you secure it onto the retrode adapter?

**Reply by kdawg8** [reply]
I got the GameSlot GG to make an adapter for someone else and I've attached a number of pictures from the build process.  The hole-to-hole soldering is a bit tricky...if I did it again I might run a wire in each hole to help the solder connect between the two pcbs.  I did have an issue where the adapter was initially shorting and the cart reader wouldn't turn on, reseating the ribbon cable fixed it.

To secure the cartridge slot to the pcb I went to a local Ace hardware store to get screws and nuts that fit, ended up getting 3/4" long 4-40 sized screws and 4-40 nuts (pictured).

![IMG_2688](https://user-images.githubusercontent.com/83562863/217454613-5b57c799-24d0-46f0-bc81-4e8b16b2b97a.jpg)
![IMG_2689](https://user-images.githubusercontent.com/83562863/217454641-dcc8c64d-8af7-44dd-8a71-e7d0ac45ec6b.jpg)
![IMG_2691](https://user-images.githubusercontent.com/83562863/217454657-3f264d34-b296-4eb3-b50c-d0051ea35ec6.jpg)
![IMG_2690](https://user-images.githubusercontent.com/83562863/217454675-b47f4015-1bfc-47ac-a0a7-cb32a8be23ff.jpg)

**Comment by SAVE-THE-HERO** [comment]
We will be able to release the Game Gear PCB data for retrosix during February.

**Comment by SAVE-THE-HERO** [comment]
Adapter for game gear.
Tested
[gg_adapter.zip](https://github.com/sanni/cartreader/files/10682708/gg_adapter.zip)
![ゲームギア01](https://user-images.githubusercontent.com/89830526/217458098-47220edc-0987-49d1-a697-2cf39acf30a5.jpg)
![ゲームギア02](https://user-images.githubusercontent.com/89830526/217458114-93b44754-feb0-406c-ba30-08351c5b686a.jpg)
![ゲームギア03](https://user-images.githubusercontent.com/89830526/217458125-fdf1d73b-8503-4efe-af0f-94f04c401e44.jpg)

**Reply by PsyK0p4T** [reply]
Nice job !
When you are using it with OSCR, what mode do you choose ? "GG Retrode" ? 
I'm currently reworking the SMS.ino code, so all adapters get my attention ;)

**Reply by SAVE-THE-HERO** [reply]
Yes, I am using RetrodeGG.
Created based on Retrode PCB wiring.
Oh, to be precise, she created it, not me.
[JL.Lab](https://blog.jllab.info/)

![IMG_4167](https://user-images.githubusercontent.com/89830526/217512364-25fd14af-206a-48de-b679-78a5ff21d58c.jpg)
![IMG_4166](https://user-images.githubusercontent.com/89830526/217512379-5b6cbc30-ebd6-4dee-85aa-dd095d713124.jpg)

**Comment by skuhn718** [comment]
Sorry to necro but this seems related. I made a TG16 adapter for the Retrode from parts I had aquired years ago from a guy in the Retrode Forums + an original TG16 cart slot... however, when I dump games to PC via the Retrode the game file appears in Windows as 'Tg16Cart.huc' and are always 1024 kb (too large) regardless of the Hu cart used.. any idea what the problem is anyone?
